File: Js/Car.js
//Befor ES6
/*
function Car(license, driver) {
  this.id;
  this.license = license;
  this.driver = driver;
  this.passenger;
}

Car.prototype.printDataCar = function () {
  console.log(this.driver)
  console.log(this.driver.name)
  console.log(this.driver.document)
}
*/
//After ES6

class Car {
  constructor(license, driver){
    this.id;
    this.license = license;
    this.driver = driver;
    this.passenger;
  }
  printDataCar() {
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
  }
}

File: Java/Car.java
class Car {
  private Integer id;
  private String license;
  private Account driver;
  private Integer passegenger;

  public Car(String license, Account driver){
      this.license = license;
      this.driver = driver;
  }

  void printDataCar() {
      if(passegenger != null){
          System.out.println("License: " + license + " Name Driver: " + driver.name + " Passengers: " + passegenger);
      }
      
  }


  public Integer getPassenger(){
      return passegenger;
  }

  public void setPassenger(Integer passenger) {
      if(passenger == 4){
          this.passegenger = passenger;
      }else{
          System.out.println("Necesitas asignar 4 pasajeros");
      }
  }

  public Integer getId() {
    return id;
  }

  public void setId(Integer id) {
    this.id = id;
  }

  public String getLicense() {
    return license;
  }

  public void setLicense(String license) {
    this.license = license;
  }

  public Account getDriver() {
    return driver;
  }

  public void setDriver(Account driver) {
    this.driver = driver;
  }


  
}